spec/controllers/ems_container_controller_spec.rb

```diff
@@ -150,10 +150,10 @@
         "default_userid"             => "",
         "default_password"           => "",
         "provider_region"            => "",
-        "metrics_selection"          => "hawkular_enabled",
-        "hawkular_security_protocol" => "ssl-without-validation",
-        "hawkular_hostname"          => "hawkular_hostname",
-        "hawkular_api_port"          => "443",
+        "metrics_selection"          => "hawkular",
+        "metrics_security_protocol"  => "ssl-without-validation",
```

Comment:
you have two spaces here:
`metrics_security_protocol"< TWO SPACIES  >=> "ssl-wit...`
their can be only one